src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -708,9 +747,8 @@ def generate_FDB_4_default_config(self, filename):
         'expectedResponseCodes': [
             [(0, ), (0,)],
```

Comment:
Please ensure that these also make sense, given the FSS locations and the CBSD locations.

Reply:
FDB.4     Used for checking responseCode of Heartbeat Response for grants 'G1' and   'G2'  (SUCCESS '0' or SUSPENDED_GRANT   '501')